include/fuzzuf/utils/random.hpp

```diff
@@ -188,15 +181,16 @@ class WalkerDiscreteDistribution {
    * @brief Construct discrete distribution from vector
    * @param (probs) Array of probabilities (weights)
    */
-  WalkerDiscreteDistribution(const std::vector<T>& probs)
-    : WalkerDiscreteDistribution(probs.cbegin(), probs.cend()) {}
+  template <class Double>
```

Comment:
This file is entirely modified by `clang-format`, but the actual diff is just this line.
The original constructor is a bit wrong. `probs` should be a vector containing the probabilities with which each element is randomly chosen. Therefore, `probs` should be something like `vector<float>` or `vector<double>`. It's not `vector<T>`.
On the contrary, `T` should be an integer type like `unsigned int` or `size_t` because `T` represents the type of a returned value, which is an index of the chosen element.